homeassistant/components/home_connect/api.py

```diff
@@ -42,6 +43,11 @@
 class ConfigEntryAuth(homeconnect.HomeConnectAPI):
     """Provide Home Connect authentication tied to an OAuth2 based config entry."""
 
+    hass: core.HomeAssistant
+    config_entry: config_entries.ConfigEntry
+    session: config_entry_oauth2_flow.OAuth2Session
+    devices: list[dict[str, Any]]
```

Comment:
You don't need to overwrite these types as these parameters are already typed

Reply:
devices type cannot be infered at initialziation, should I set the type at `__init__` (what it was already) or should I define it here?